src/snyk/snykCode/codeService.ts

```diff
@@ -195,12 +197,12 @@ export class SnykCodeService extends AnalysisStatusProvider implements ISnykCode
   }
 
   private async isEnabled(): Promise<boolean> {
-    const settings = await getSastSettings();
+    const settings = await getSastSettings(this.snykApiClient);
```

Comment:
Same here, would be better, to inject this on construction.

Reply:
See the answer in `user.ts`

Reply:
There's no class definition for getSastSettings method. Since class definition is not really necessary for a single method in this case, I'd prefer to leave it as is for now.